stock_picking_show_date_done/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 PlanetaTIC - Lluís Rovira <lrovira@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Stock Picking Show Date Done',
+    'version': '10.0.1.0.0',
+    'category': 'Warehouse Management',
+    'author': 'PlanetaTIC, Odoo Community Association (OCA)',
+    'website': 'https://www.planetatic.com',
```

Comment:
Please follow this template example : https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py